docs/apps/transfer/messages.md

```diff
@@ -31,7 +31,26 @@ This message is expected to fail if:
 - `Sender` is empty.
 - `Receiver` is empty.
 - `TimeoutHeight` and `TimeoutTimestamp` are both zero.
+- `Memo` is longer than 256 characters.
```

Comment:
I don't think this is true?

Reply:
Oh wow, you're right. But it is true for ICA. I'll fix. https://github.com/cosmos/ibc-go/blob/ec3166d31cd86f138e1c5559f29483c348e0b901/modules/apps/27-interchain-accounts/types/packet.go#L49

Reply:
Removed this line